protocols/relay/src/v2.rs

```diff
@@ -30,6 +30,13 @@ mod copy_future;
 mod protocol;
 pub mod relay;
 
+pub use protocol::{
+    inbound_hop::FatalUpgradeError as InHopUpgradeError,
```

Comment:
```suggestion
    inbound_hop::FatalUpgradeError as InboundHopFatalUpgradeError,
```

While lengthy, I would prefer writing these out for the sake of being easier to understand.